UnityProject/Assets/Scripts/Managers/PlayerList.Admin.cs

```diff
@@ -423,18 +425,34 @@ private async Task<bool> CheckUserState(string unverifiedToken, ConnectedPlayer
 			}
 		}
 
-		//Whitelist checking:
-		var lines = File.ReadAllLines(whiteListPath);
-
-		//Checks whether the userid is in either the Admins or whitelist AND that the whitelist file has something in it.
-		//Whitelist only activates if whitelist is populated.
-		if (lines.Length > 0 && !adminUsers.Contains(userId) && !whiteListUsers.Contains(userId))
+		if (isAdmin == false)
```

Comment:
This would prevent mentors from joining a full server, possibly use a mentorUsers.contains check here too.

Reply:
as a sidenote i'll need to remember to change this check to a specific permission whenever/if i get around to the admin refactor

Reply:
They are players just like the rest with no important responsibilities. They don't need to skip the line.